src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -871,7 +871,15 @@ public static string MapToSnapshotNotSupportedWithoutContext(string snapshotType
 
          public const string UnableToLoadQualificationConfigurationFromOptions = "Unable to read configuration from options";
 
+         public const string QualificationSnapshotFileNotDefined = "Qualification snapshot file not defined.";
```

Comment:
@Yuri05 Because those messages are what will be visible to the user in the CLI, they should be as explicit as possible.
This is just a placeholder for now. I am going to create a new issue to review all those configuration messages

Reply:
Could you place a TODO in all placeholder-messages, so that we do not forget to finalize them?

Reply:
when TODOs are added, we can merge this